conv_model_fwd.cpp

```diff
@@ -128,8 +128,33 @@ int conv_benchmark(int argc, char** argv) {
     tensor_copy_KCRS_to_KCRSck(naive_filter, (float*)filter_libxsmm, K, C, R, S, bc, bk);
   }
   
+  long avoid_rim_fmas = 0;
+  if (ofh <= 7 && ofw <=7 && R == 3 && S == 3 && stride_w == 1 && stride_h == 1 && h_in_gemm == 1) {
+    avoid_rim_fmas = 1;
+  }
+
+  if (R != 1 || S != 1) {
+    pack_input = 0;
+  }
+
+  long Cb_step = Cb/c_block;
```

Comment:
Had to shuffle this up in the code to have Cb_step, r_step and s_step defined before l_flags (where we put BETA_) flag conditionally now)